### ISSUE TITLE
format of bunch code of netrw after #16248 looks mess

### ISSUE THREAD

**Issue by Shane-XB-Qian** [vim/vim#16263]
### Steps to reproduce

i saw many tickets/mails recently from you about code format and changing default etc,
but wish your PR was correct and result was checked firstly, thx.
// perhap better revert that PR first, then resubmit if like to, 
// otherwise the git history maybe messy later, since bunch code changed.

### Expected behaviour

none.

### Version of Vim

v9.1.945

### Environment

linux

**Comment by Shane-XB-Qian** [comment]
#16248 @saccarosium

**Comment by Shane-XB-Qian** [comment]
bunch code after `netrw#Open()`

**Comment by Shane-XB-Qian** [comment]
[foo.patch.txt](https://github.com/user-attachments/files/18208447/foo.patch.txt)
example patch as foo.patch, but maybe better revert, then re-think to submit, or the git history maybe messy.

**Comment by zeertzjq** [comment]
Ah, it seems `endf` isn't indented correctly.

**Comment by Shane-XB-Qian** [comment]
yes, and bunch code after that unnecessary indented, 

and E.g:
```vim
" Git Bash
if has('win32unix')
  " (cyg)start suffices
  let s:os_viewer = ''
  " Windows / WSL
elseif executable('explorer.exe')
  let s:os_viewer = 'explorer.exe'
  " Linux / BSD
elseif executable('xdg-open')
  let s:os_viewer = 'xdg-open'
  " MacOS
elseif executable('open')
  let s:os_viewer = 'open'
endif
```
~~why 3 spaces here, if like to format, then shouldnot do it better?~~
UPD: sorry, it is more about the comment above if/else, maybe better align comment to if/else header.

**Comment by saccarosium** [comment]
I simply run `=G` on the top of the file. I didn't really checked everything. Glad you have made a PR to address any issues

**Comment by chrisbra** [comment]
I suppose this is fixed now witht #16266
